pkg/gatewayserver/io/ws/lbslns/downstream.go

```diff
@@ -154,15 +152,15 @@ func (*lbsLNS) TransferTime(ctx context.Context, serverTime time.Time, conn *io.
 		return nil, nil
 	}
 
-	_, _, _, np, n := conn.RTTStats(transferTimeRTTPercentile, serverTime)
+	_, _, median, _, n := conn.RTTStats(100, serverTime)
```

Comment:
We are no longer accepting outliers? What is median about p100?
Isn't this basically considering the max value?

Reply:
The percentile parameters only affects `np`, the last returned element in the tuple, right ?

https://github.com/TheThingsNetwork/lorawan-stack/blob/bb54ef7af244e85b55c5311e64f41376fffbe919/pkg/gatewayserver/io/rtts.go#L63-L92

Reply:
Ah right I missed that here, thanks